R/report_participants.R

```diff
@@ -1,18 +1,22 @@
 #' Reporting the participant data
 #'
-#' A helper function to help you format the participants data (age, sex, ...) in the participants section.
+#' A helper function to help you format the participants data (age, sex, ...) in
+#' the participants section.
 #'
 #' @param data A data frame.
-#' @param age The name of the column containing the age.
-#' @param sex The name of the column containing the sex. Note that classes
-#'   should be one of `c("Male", "M", "Female", "F")`.
+#' @param age The name of the column containing the age of the participant.
+#' @param sex The name of the column containing the sex of the participant. The
```

Comment:
Though I suppose we could fix this behaviour?

Reply:
What approach were you thinking?

I was thinking we can check unique number of levels for `Sex` variable and then:
- if <= 2, report % females
- else, report % females and % males

What do you think?